code/game/gamemodes/cult/runes.dm

```diff
@@ -167,10 +172,6 @@ structure_check() searches for nearby cultist structures required for the invoca
 
 /obj/effect/rune/malformed/invoke(var/list/invokers)
 	..()
-	for(var/M in invokers)
-		var/mob/living/L = M
-		to_chat(L, "<span class='cultitalic'><b>You feel your life force draining. The Geometer is displeased.</b></span>")
```

Comment:
Nope, but it's in the code as a base type in case something goes horribly wrong.